packages/docusaurus-plugin-google-gtag/src/types.d.ts

```diff
@@ -10,11 +10,7 @@
 interface Window {
   gtag: (
     command: string,
-    fields: string,
-    params: {
-      page_title?: string;
-      page_location?: string;
-      page_path?: string;
-    },
+    action: string,
+    params?: string | Gtag.EventParams,
   ) => void;
```

Comment:
do we still need this TS if we have gtag.js typedefs? Why?

This dependency is supposed to provide all the types we need

Have you tried using my recos like the /// directive?

Looks similar to what someone suggest here: 
https://andrew-simpson-ross.medium.com/strongly-typed-google-analytics-v4-with-next-js-aad6c6a5e383

Reply:
I'm not sure but probably not. I don't understand the intricacies of typedefs and the original purpose of this file